weather-project/src/utility/Utility.js

```diff
@@ -0,0 +1,96 @@
+export const dateConverter = (date) => {
+  const daysWeek = [
+    "Sonntag",
+    "Montag",
+    "Dienstag",
+    "Mittwoch",
+    "Donnerstag",
+    "Freitag",
+    "Samstag",
+  ];
+
+  const month = [
+    "Januar",
+    "Februar",
+    "März",
+    "April",
+    "Mai",
+    "Juni",
+    "Juli",
+    "August",
+    "September",
+    "Oktober",
+    "November",
+    "Dezember",
+  ];
+
+  const day = date.getDate();
+  const dayNumber = date.getDay();
+  const monthNumber = date.getMonth();
+
+  const dayDate = daysWeek[dayNumber];
+  const monthDate = month[monthNumber];
+
+  return { day, dayDate, monthDate };
+};
+
+export const timeConverter = (date) => {
+  const hours = date.getHours();
+  const minutes = date.getMinutes();
+  return {hours, minutes}
+}
+
+export const tempConverter = (temp, unit = "Kelvin") => {
+  if ((unit = "Kelvin")) {
+    const convertedTemp = (temp - 273.15).toFixed(1);
+    return convertedTemp;
```

Comment:
Hier kannst du auch direkt convertedTemp returnen, also 
return (temp- 273.15).toFixed(1); 
in diesem Beispiel